benchmark/conftest.py

```diff
@@ -1,29 +1,28 @@
 import os
 
 import requests
+from dotenv import load_dotenv
```

Comment:
Why do we need `dotenv`? The implementation seems a bit hacky ;)

Reply:
Might be a bit of a work around, but this was the only way we could implement the key loading. We have some questions, later per mail

Reply:
@ytehran could you or another team member address this? prevents me from merging.